resources/views/utils/user/profile-picture.blade.php

```diff
@@ -24,28 +24,28 @@
             @csrf
             <div class="card-action valign-center">
                 <x-input.file s="12" l="7" xl="8" id="picture" style="margin-top:auto" accept=".jpg,.png,.jpeg"
-                                text="general.browse" required/>
+                                text="general.browse" required helper=".jpg,.png,.jpeg fájlok tölthetőek fel, maximum 2 MB-os méretig."/>
```

Comment:
Same here.

Reply:
I updated the file component. you need to reset cache.